pkg/plugins/policies/meshtrace/api/v1alpha1/meshtrace.pb.go

```diff
@@ -95,6 +95,12 @@ type MeshTrace_DatadogBackend struct {
 	Address string `protobuf:"bytes,1,opt,name=address,proto3" json:"address,omitempty"`
 	// Port of datadog collector
 	Port uint32 `protobuf:"varint,2,opt,name=port,proto3" json:"port,omitempty"`
+	// Determines if datadog service name should be split based on traffic
+	// direction and destination. For example, with `splitService: true` and a
+	// `backend` service that communicates with a couple of databases, you would
+	// get service names like `backend_INBOUND`, `backend_OUTBOUND_db1`, and
+	// `backend_OUTBOUND_db2` in Datadog. Default: false
+	SplitService bool `protobuf:"varint,3,opt,name=splitService,proto3" json:"splitService,omitempty"`
```

Comment:
Mirrored from current TrafficTrace, should I add it to MADR as well or is it ok to leave it out?

Reply:
no strong opinion here, closing